packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/android/signingout.md

```diff
@@ -0,0 +1,43 @@
+This code assumes that the access token is stored after a successful sign-in:
+
+```kotlin
+fun logout() {
```

Comment:
This code isn't really relevant to the IDX SDK, we should punt revoke functionality to being documented with [okta-mobile-kotlin](https://github.com/okta/okta-mobile-kotlin).

Reply:
I agree that it's not part of the IDX SDK, however documenting sign-in implicitly includes documenting sign-out. Although the way to do that and the code are included in other parts of the documentation, there's explicit OIE version of a "sign users out" Guide (the current one is for Classic Engine.)

The goal of this guide is to be the single source of truth for the basics of using the mobile IDX SDKs. Other guides and samples can refer to this guide for particular tasks, such as sign out a user, instead of recreating something.

Unless there's strong reasons to the contrary, I'll keep this section.